includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -689,12 +689,16 @@ public function get_charge( $charge_id ) {
 	/**
 	 * List disputes
 	 *
+	 * @param  int $page The page index to retrieve.
+	 * @param  int $page_size The number of items the page contains.
 	 * @return array
 	 * @throws API_Exception - Exception thrown on request failure.
 	 */
-	public function list_disputes() {
+	public function list_disputes( $page = 0, $page_size = 25 ) {
```

Comment:
I believe we can make the parameters/return type strictly typed here.

Reply:
Done thanks